src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
 
+        // This represents how much EC to consume per executed instruction.
+        // This would be the "variable" component of the processor's power.
+        // TODO: This value needs to be changed to a non-zero number and balanced.
```

Comment:
The todo is part of this PR.  It needs to be done before merging.  But I didn't want to presume to change it myself without showing it to every one else first.